Chaincase.Common/Extensions.cs

```diff
@@ -43,7 +44,7 @@ public static IServiceCollection AddCommonServices(this IServiceCollection servi
                 var bitcoinStore = x.GetRequiredService<BitcoinStore>();
 
                 if (config.UseTor)
-                    return new ChaincaseSynchronizer(network, bitcoinStore, () => config.GetCurrentBackendUri(), config.TorSocks5EndPoint);
+                    return new ChaincaseSynchronizer(network, bitcoinStore, () => new Uri("http://tgkjgdbmqellj6yfzmmwa4nqk32eezkeauqmw7tr63t7i2sdaetyrvyd.onion/"), config.TorSocks5EndPoint);
```

Comment:
😾

Reply:
This is here cause switching this value in config from already synced app is not supported in the UI